torch/nn/modules/conv.py

```diff
@@ -323,7 +323,7 @@ def __init__(self, in_channels, out_channels, kernel_size, stride=1,
                  bias=True, padding_mode='zeros'):
         kernel_size = _pair(kernel_size)
         stride = _pair(stride)
-        padding = _pair(padding)
+        padding = padding if isinstance(padding, str) else _pair(padding)
```

Comment:
`isinstance` metacompilation will only work in an if statement currently, not ternary expression